components/rewards/PointsHistory.js

```diff
@@ -32,4 +33,11 @@ function PointsHistory({ transactions, user, updates, navigation }) {
   return <View />;
 }
 
+PointsHistory.propTypes = {
+  transactions: PropTypes.object.isRequired,
+  user: PropTypes.object.isRequired,
```

Comment:
This gets updated in #48 (sorry), but the linter should show you that only `transactions` is actually being referenced (the others are defined and never used), so you don't need `user`, `updates`, or `navigation`

Reply:
@kennethlien remember to remove the (now-missing) variables